### DISCUSSION TITLE
Way to perform additional smoothing for tissues besides scalp

### DISCUSSION THREAD

**Discussion by divyarajasekharan** [simnibs/simnibs#382] [7.Creating head models]
Hello! 

I am generating head models with only T1w images available, so the skull boundary is not always well segmented. We're able to perform manual corrections, but this sometimes results in a CSF mesh that contains many ridges: 

![CSF](https://github.com/simnibs/simnibs/assets/91343323/dfae7d3f-2338-4c84-9b39-df11f2fe958a)

Is there any way to perform additional smoothing steps for the outer CSF surface only, similar to how it's done for the scalp? Thanks!

**Comment by Kstoffer** [comment]
Could a smoothing operation on the volume representation (nifti) of the skull only prior to meshing possibly resolve the issue?
It is unfortunately not trivial to smooth inner surfaces while maintaining tetrahedral quality.
